docs/en/faq/common-issues-in-deployment-and-development-phase.md

```diff
@@ -253,3 +253,55 @@ location /apollo/ {
     proxy_pass http://127.0.0.1:8070/;
 }
 ```
+
+### 17. How to configure https after Portal is mounted to nginx/slb?
+
+Firstly, configure https access configuration on nginx/slb, taking nginx as an example:
+
+```
+     server {
+         # If the nginx version is lower and does not support http2, configure listen 443 ssl;
+         listen 443 ssl http2;
+         server_name your-domain-name;
+         # ssl certificate, nginx needs to use a certificate with a complete certificate chain
+         ssl_certificate /etc/nginx/ssl/xxx.crt;
+         ssl_certificate_key /etc/nginx/ssl/xxx.key;
+
+         location / {
+             proxy_pass http://apollo-portal-address:8070;
+             proxy_set_header x-real-ip $remote_addr;
+             proxy_set_header x-forwarded-for $proxy_add_x_forwarded_for;
+             #! ! ! This must be $http_host, if it is configured as $host, the port will be wrong when redirecting
+             proxy_set_header host $http_host;
+             proxy_set_header x-forwarded-proto $scheme;
+             proxy_http_version 1.1;
+         }
+     }
+```
+
+Secondly, configure the http redirection strategy, and choose one of the following two methods.
+
```

Comment:
the `server.forward-headers-strategy` is not optional but required.
nginx/slb redirect cannot work correctly without the `server.forward-headers-strategy`, it may cause some url located to the `http://apollo-portal-domain/some-resource` and then blocked by the browser

https://www.w3.org/TR/mixed-content/#should-block-fetch

|-|only `server.forward-headers-strategy`|only `nginx/slb redirect`| both configured |
|---|---|---|---|
|http request|http response|redirect to https|redirect to https|
|https request|https response|https response first, and then may be a failure caused by mixed-content|https response|

Reply:
Thanks for pointing this out. This is a situation I missed.